lib/bibliothecary/parsers/nuget.rb

```diff
@@ -47,6 +51,30 @@ def self.parse_project_lock_json(file_contents)
         end
       end
 
+      def self.parse_packages_lock_json(file_contents)
+        manifest = JSON.parse file_contents
+
+        frameworks = {}
+        manifest.fetch('dependencies',[]).each do |framework, deps|
+          frameworks[framework] = deps.map do |name, details|
+            {
+              name: name,
+              # I think 'resolved' should always be set though
```

Comment:
Do we have any rules about "I think"s in comments?

Reply:
change it to "I hope"?

Reply:
nooo i mean like, "I" cus then we gotta git annotate to see who I is